components/skus/renderer/skus_js_handler.cc

```diff
@@ -44,7 +48,7 @@ bool SkusJSHandler::EnsureConnected() {
   }
   bool result = skus_service_.is_bound();
 #if BUILDFLAG(ENABLE_BRAVE_VPN)
-  if (!vpn_service_.is_bound()) {
+  if (brave_vpn::IsBraveVPNFeatureEnabled() && !vpn_service_.is_bound()) {
```

Comment:
```
if (brave_vpn::IsBraveVPNFeatureEnabled()) {
  if (!vpn_service_.is_bound()) {
    render_frame_->GetBrowserInterfaceBroker()->GetInterface(
        vpn_service_.BindNewPipeAndPassReceiver());
  }
  result = result && vpn_service_.is_bound();
}
```

Reply:
I think this should fix https://github.com/brave/brave-browser/issues/35508

Reply:
Perfect! Thanks for catching 👍🏼 Fixed.